<launch>
  <node type="nodelet" pkg="nodelet" name="fisheye_manager" args="manager" output="screen"/>
  <node type="nodelet" pkg="nodelet" name="fisheye_to_panorama"
        args="load jsk_perception/FisheyeToPanorama fisheye_manager" output="screen">
    <remap from="~input" to="/camera/image_raw"/>
    <param name="use_panorama" value="false"/>
    <param name="k" value="275.0"/>
    <param name="odom_topic_name" value="/dji_sdk/odometry" />
  </node>

  <node type="image_view" pkg="image_view" name="fisheye_image_view">
    <remap from="image" to="/camera/image_raw"/>
  </node>

  <node type="image_view" pkg="image_view" name="undistorted_image_view">
    <remap from="image" to="/fisheye_to_panorama/output"/>
  </node>
</launch>
